themes/bootstrap5/templates/primo/cites.phtml

```diff
@@ -1,4 +1,3 @@
 <?php
 $this->pageTitle = 'results_cited_by_title';
-$this->pageTitleLink = 'results_cited_by_title_link_html';
```

Comment:
I don't think this translation string is used anymore; should we consider deleting it?